src/__mocks__/changeStatus.js

```diff
@@ -0,0 +1,13 @@
+import localStorageMock from './localStorage';
+
+const changeStatus = (arr, index) => {
+  if (arr[index].completed) {
+    arr[index].completed = false;
+  } else {
+    arr[index].completed = true;
+  }
+  localStorageMock.setItem('data', arr);
+  return arr;
+};
+
+export default changeStatus;
```

Comment:
- [ ] As mentioned in the previous comment the mock testing it can be done using the jest documentation.

Reply:
Hi, thanks for reviewing. Could you be more specific about what we have done incorrectly here?